db/migrate/20230405140520_change_imageurl_from_products.rb

```diff
@@ -0,0 +1,13 @@
+class ChangeImageurlFromProducts < ActiveRecord::Migration[7.0]
```

Comment:
is this an irreversible migration?

Reply:
Sorry sir I misread the question. I have made the correction.